editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2708,32 +2708,99 @@ void Node3DEditorViewport::_draw() {
 				handle_color,
 				Math::round(2 * EDSCALE));
 	}
-	if (previewing) {
-		Size2 ss = Size2(ProjectSettings::get_singleton()->get("display/window/size/width"), ProjectSettings::get_singleton()->get("display/window/size/height"));
-		float aspect = ss.aspect();
-		Size2 s = get_size();
+	if (previewing && EDITOR_GET("editors/3d/show_camera_preview_guides")) {
```

Comment:
This means that if the thirds guides are disabled, there's also no aspect ratio rectangle, right?

Reply:
Yes, they're toggled both at the same time.